src/main/java/org/hydev/mcpm/client/database/LocalPluginTracker.java

```diff
@@ -276,10 +277,10 @@ public String getVersion(String name) {
         return "";
     }
 
-    /**
-     * Compares the hash of the locally-installed plugin of a specified 
-     * version with the hash of the plugin of that version on the server
-     * @return True if the hashes match, false otherwise
+    /*
```

Comment:
Is this the right format for docstrings? It probably doesn't make much of a difference, but just making sure.

Reply:
This isn’t my code idk why it’s here how do I sync it with main

Reply:
It's not the right format for docstrings, but the function below is commented out so having a docstring on nothing gives checkstyle errors so Peter changed it. It's fine